internal/website/content/howto/blob/_index.md

```diff
@@ -5,29 +5,64 @@ showInSidenav: true
 toc: true
 ---
 
-Blobs are a common abstraction for storing unstructured data on cloud storage
-providers and accessing them via HTTP. These guides show how to work with
+Blobs are a common abstraction for storing unstructured data on Cloud storage
+services and accessing them via HTTP. These guides show how to work with
 blobs in the Go CDK.
 
 <!--more-->
 
+The Go CDK supports operations like reading and writing blobs (using standard
+[io package][] interfaces), deleting blobs, and listing blobs in a bucket.
+
+Subpackages contain driver implementations of blob for various services,
+including Cloud and on-prem solutions. You can develop your application
+locally using [`fileblob`][], then deploy it to multiple Cloud providers with
+minimal reconfiguration of your initialization code.
+
+[io package]: https://golang.org/pkg/io/
+[`fileblob`]: https://godoc.org/gocloud.dev/blob/fileblob
+
 ## Opening a Bucket {#opening}
 
-The first step in interacting with unstructured storage is connecting to your
-storage provider. Every storage provider is a little different, but the Go CDK
-lets you interact with all of them using the [`*blob.Bucket` type][].
+The first step in interacting with unstructured storage is opening a connection
+to your storage service to instantiate a portable [`*blob.Bucket`][].
 
 The easiest way to open a blob is using [`blob.OpenBucket`][] and a URL
 pointing to the blob, making sure you ["blank import"][] the driver package to
-link it in. See [Concepts: URLs][] for more details. If you need
+link it in. Do not use `NewBucket` function, intended for use by the driver
+implementations only. See [Concepts: URLs][] for more details.
```

Comment:
Maybe "for more background on how URLs are used in the Go CDK"? Otherwise it sounds like that's where you should go for more details on what the service-specific URLs look like. Maybe move this sentence after the example? And/or, extend it to include a link to below, e.g.,

"See the [guide below][] for URL forms for supported services, and [Concepts: URLs][] for more background on how URLs are used in the Go CDK."

Reply:
Done

I think it would be a good idea to add a table of API/url schemas to the concepts doc (in a different PR).